code/modules/mob/mob_grab.dm

```diff
@@ -332,7 +334,84 @@
 						dislocate(affecting, assailant, hit_zone)
 
 				if(I_DISARM)
-					pin_down(affecting, assailant)
+					if(hit_zone == "chest")
+						pin_down(affecting, assailant)
+					var/mob/living/carbon/human/H = affecting
+					var/hairstyle = H.h_style
+					var/obj/item/weapon/grab/G = assailant.get_active_hand()

```

Comment:
pretty sure get_active_hand just returns the item in the hand, doesn't necessarily mean a grab - so this is unsafe.

I might be wrong and talking out of my ass though.

Reply:
I don't really know why you need to get a ref to the grab at all, considering that this the grab item's attack() proc.